mod_reforged/hooks/entity/tactical/humans/nomad_slinger.nut

```diff
@@ -36,6 +36,6 @@
 
 		// Reforged
 		this.m.Skills.add(::new("scripts/skills/perks/perk_rotation"));
-		this.m.Skills.add(::new("scripts/skills/perks/perk_rf_target_practice"));
+		this.m.Skills.add(::new("scripts/skills/perks/perk_rf_small_target"));
```

Comment:
Ah this is going to hurt lol. I'd say give them Bullseye instead, for the added armor penetration against unconvered targets. It also kinda thematically makes sense to "lob" the stones over at covered targets behind their cover.

Reply:
My primary concern was that Slings apply Dazed on a headshot, which lasts 2 turns. Giving them more headshot chance will increase the frequency of that which may be too much for the player.

Reply:
vc

Reply:
We decided to add Calculated Strikes instead.

Reply:
Resolved by giving calculated strikes instead of small target